packages/common/src/components/AppHeader/AppHeaderTabs.tsx

```diff
@@ -39,9 +64,10 @@ const AppHeaderTabs: React.FC<Props> = ({ tabs }) => {
           <a
             tabIndex={0}
```

Comment:
Why do we have `tabIndex={0}` on a link? The should be "navigatable" by default

Reply:
Maybe it has something to do with the UX-components used but without the tabIndex the tabs are not focusable.